mrmustard/lab_dev/circuit_components_utils.py

```diff
@@ -97,3 +96,30 @@ def __init__(
     @property
     def representation(self) -> Bargmann:
         return Bargmann(*triples.displacement_map_s_parametrized_Abc(self.s, len(self.modes)))
+
+
+class _BtoQMap(CircuitComponent):
+    r"""The change of representation kernel from Bargmann into Quadrature as a ``Unitary``.
```

Comment:
Can we remove the underscores from here and Dsmap? I don't think there's a need to keep them private, since they're in this file which is very clear about containing unphysical maps